lib/modules/manager/circleci/extract.ts

```diff
@@ -58,13 +58,11 @@ export function extractPackageFile(
 ): PackageFileContent | null {
   const deps: PackageDependency[] = [];
   try {
-    const parsed = parseSingleYaml(content, {
-      customSchema: CircleCiFile,
-    });
+    const parsed = CircleCiFile.parse(parseSingleYaml(content));
```

Comment:
`customSchema` parameter breaks type inference, marking the `aliases` field as `unknown`.